domainmodel/2.23.0/org.eclipse.xtext.example.domainmodel.releng/pom.xml

```diff
@@ -34,7 +34,7 @@
 		<moduleProperties></moduleProperties>
 		<systemProperties></systemProperties>
 		<tycho.testArgLine></tycho.testArgLine>
-		<xtextVersion>${project.version}</xtextVersion>
+		<xtextVersion>2.23.0</xtextVersion>
```

Comment:
why got this replaced?

Reply:
cause the project version is 2.23.0-SNAPSHOT cause some @kthoms did not want to change it to 1.0.0-SNAPSHOT as it is for wizard examples

Reply:
ok